src/5B320.c

```diff
@@ -3,6 +3,175 @@
+#define MSG_ItemName_SmashCharge0 0
+#define MSG_ItemName_SuperFocus 0
+#define MSG_ItemName_SuperJump 0
+#define MSG_ItemName_SuperSmash 0
+#define MSG_ItemName_SuspiciousNote 0
+#define MSG_ItemName_ToadDoll 0
+#define MSG_ItemName_UnusedLetter 0
+#define MSG_ItemName_UnusedLetter02 0
+#define MSG_ItemName_UnusedLetter03 0
+#define MSG_ItemName_UnusedLetter04 0
+#define MSG_ItemName_UnusedLetter05 0
+#define MSG_ItemName_UnusedLetter06 0
+#define MSG_ItemShortDesc_AutoJump 0
+#define MSG_ItemShortDesc_AutoMultibounce 0
+#define MSG_ItemShortDesc_AutoSmash 0
+#define MSG_ItemShortDesc_Berserker 0
+#define MSG_ItemShortDesc_CrazyHeart 0
+#define MSG_ItemShortDesc_EarthquakeJump 0
+#define MSG_ItemShortDesc_FlowerFanatic 0
+#define MSG_ItemShortDesc_ForestPass 0
+#define MSG_ItemShortDesc_HealthyHealthy 0
+#define MSG_ItemShortDesc_HustleDrink 0
+#define MSG_ItemShortDesc_InsecticideHerb 0
+#define MSG_ItemShortDesc_JumpCharge0 0
+#define MSG_ItemShortDesc_Kaiden 0
+#define MSG_ItemShortDesc_KootAltPhoto 0
+#define MSG_ItemShortDesc_KootShell 0
+#define MSG_ItemShortDesc_MegaHPDrain 0
+#define MSG_ItemShortDesc_MysteryScroll 0
+#define MSG_ItemShortDesc_PleaseComeBack 0
+#define MSG_ItemShortDesc_Screwdriver 0
+#define MSG_ItemShortDesc_ShellCrack 0
+#define MSG_ItemShortDesc_SmashCharge0 0
+#define MSG_ItemShortDesc_SuperFocus 0
+#define MSG_ItemShortDesc_SuperJump 0
+#define MSG_ItemShortDesc_SuperSmash 0
+#define MSG_ItemShortDesc_SuspiciousNote 0
+#define MSG_ItemShortDesc_ToadDoll 0
+#define MSG_ItemShortDesc_UnusedLetter 0
+#define MSG_ItemShortDesc_UnusedLetter02 0
+#define MSG_ItemShortDesc_UnusedLetter03 0
+#define MSG_ItemShortDesc_UnusedLetter04 0
+#define MSG_ItemShortDesc_UnusedLetter05 0
+#define MSG_ItemShortDesc_UnusedLetter06 0
+#define MSG_MoveName_32 0
+#define MSG_MoveName_AutoJump 0
+#define MSG_MoveName_AutoMultibounce 0
+#define MSG_MoveName_AutoSmash 0
+#define MSG_MoveName_Berserker 0
+#define MSG_MoveName_BumpAttack 0
+#define MSG_MoveName_CloseCall 0
+#define MSG_MoveName_CrazyHeart 0
+#define MSG_MoveName_DefendPlus 0
+#define MSG_MoveName_DizzyAttack 0
+#define MSG_MoveName_EarthquakeJump 0
+#define MSG_MoveName_FlowerFanatic 0
+#define MSG_MoveName_FlowerFinder 0
+#define MSG_MoveName_HealthyHealthy 0
+#define MSG_MoveName_HeartFinder 0
+#define MSG_MoveName_ISpy 0
+#define MSG_MoveName_JumpCharge0 0
+#define MSG_MoveName_LastStand 0
+#define MSG_MoveName_LuckyDay 0
+#define MSG_MoveName_MegaHPDrain 0
+#define MSG_MoveName_PowerRush 0
+#define MSG_MoveName_SmashCharge0 0
+#define MSG_MoveName_SpeedySpin 0
+#define MSG_MoveName_SpinAttack 0
+#define MSG_MoveName_SuperJump 0
+#define MSG_MoveName_SuperSmash 0
+#define MSG_PartnerDesc_Goombaria_Battle 0
+#define MSG_PartnerDesc_Goombaria_World 0
+#define MSG_PartnerDesc_Goompa_Battle 0
+#define MSG_PartnerDesc_Goompa_World 0
+#define MSG_PartnerDesc_Twink_Battle 0
+#define MSG_PartnerDesc_Twink_World 0
+#define MSG_ShortDesc_32 0
+#endif
+
+#if !defined(VERSION_JP)
```

Comment:
Why not combine these blocks?

Reply:
I pulled them apart because they have different goals.